source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -1,33 +1,38 @@
 3rd Party Libraries
 ===================
 
-Teams that are not using parts provided in the KoP will likely need to install external dependencies. Please view the below section to see information on adding an external library.
+Teams that are using non-PWM motor controllers or advanced sensors will most likely need to install external vendor dependencies. Please view the below section to see information on adding an external library.
 
-The Mechanism
--------------
+What Are Vendor Dependencies?
+-----------------------------
 
-In support of this effort NI (for LabVIEW) and FIRST/WPI (for Java/C++) have developed mechanisms that should make it easy for vendors to plug their code into the WPILib software and for teams to use that code once it has been installed. A brief description of how the system works for each language can be found below.
+A vendor dependency is a way for vendors such as CTRE, REV, and others to add their own custom code that interfaces with motor controllers and other devices. This way teams can interact with their devices via CAN and have access to more complex and in-depth features than traditional PWM control.
 
-The Mechanism - Java/C++
-^^^^^^^^^^^^^^^^^^^^^^^^
+Managing Vendor Dependencies
+----------------------------
+
+Vendor dependencies are installed on a per-project basis (so each robot project can have its own set of vendor dependencies). Vendor dependencies can be installed "online" or "offline". The "online" functionality is done by downloading the dependencies over the internet, while offline is typically provided by a vendor-specific installer.
```

Comment:
I think the recommendation down below to use offline installers might be better here.